packages/record-tuple-polyfill/src/tuple.js

```diff
@@ -37,7 +34,7 @@ function createFreshTupleFromIterableObject(value) {
 
     const tuple = Object.create(Tuple.prototype);
     // eslint-disable-next-line no-constant-condition
-    for (const value of unboxed) {
+    for (const value of value) {
```

Comment:
This error isn't caught by the tests: is this function ever executed?

Reply:
What error are you referring to?

Reply:
When I renamed `unboxed` to `value`, it should have thrown a TDZ error (because I'm iterating over `values` which at that point is shadowed and not initialized yet):

```js
let value = [];
for (const value of value);
```

However, as you can see CI passed for the previous commit even with this bug I introduced.

Reply:
Oh I see. The function is certainly called by the tests, all tuples flow through this method. I'm not entirely sure why the CI didn't fail on that.